snuba/query/columns.py

```diff
@@ -41,6 +42,9 @@ def column_expr(dataset, column_name, query: Query, parsing_context: ParsingCont
     elif isinstance(column_name, str) and QUOTED_LITERAL_RE.match(column_name):
         return escape_literal(column_name[1:-1])
     else:
+        # Handle column alias
+        if is_alias_column_expr(column_name):
+            (column_name, _, alias) = column_name
```

Comment:
what is the expected behavior for columns where we generate the alias internally (like for tags) ?
example, what should this resolve to ?
```
['tags[level]', None, 'my_name_for_this_col']
```